src/atomate2/common/utils.py

```diff
@@ -11,6 +11,8 @@
 if TYPE_CHECKING:
     from pathlib import Path
 
+PHONON_SYM_PREC = 1e-4
```

Comment:
Do you think this should be defined in the `SETTINGS` object?

Reply:
As an additional symprec or as a standard symprec? I also think we might have to test this setting on a larger number of structures.

Reply:
I would imagine this should be an additional symprec.